### PR TITLE
✨Source Open Exchange Rates: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #30906: :sparkles: Source Open Exchange Rates : Migrate Python CDK to Low-code CDK
https://docs.airbyte.com/integrations/sources/open-exchange-rates

Rules for the Connectors Hackathon
You can only work on one connector at a time
Make sure to resolve all suggestions from the Airbyte team
Must include all streams of the connector you’re migrating. Otherwise, it will not be accepted.
If you don’t commit anything within five days, the issue may be assigned to someone else.

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Open Exchange Rates to Low-Code CDK*
Closes #30906

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results
image

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Open Exchange Rates</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Test results are a little stumbled upon incremental reads, but full refresh is passing
image

**Comment by btkcodedev** [comment]
config
{
  "start_date": "2023-08-01",
  "base": "USD",
  "app_id": "0f492965c4e24af7bc1f5a6ddc63acbb"
}

Open-temp-creds

**Comment by btkcodedev** [comment]
Ready for review comments please :)

**Comment by marcosmarxm** [comment]
Thanks for the contribution @btkcodedev. I added it to the team backlog and we're planning to start next week the review process.

File: airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
 connector_image: airbyte/source-open-exchange-rates:dev
-
 acceptance_tests:
   spec:
     tests:
       - spec_path: "source_open_exchange_rates/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: 0.1.0
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml:9]: If the previous version isn't backward compatible you need to release a major version, change to 1.0.0 as a breaking change. Why are you update the schema and specification of this connector?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left a review to understand the breaking change.

**Comment by btkcodedev** [comment]
Oops, that was unwanted, Fixed the schema and reverted that skipping of test :)
New test result: All pass except inc sync

image

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**